nucliadb/nucliadb/ingest/processing.py

```diff
@@ -440,3 +424,58 @@ async def send_to_process(
         return ProcessingInfo(
             seqid=seqid, account_seq=account_seq, queue=QueueType(queue_type)
         )
+
+
+class DummyProcessingEngine(ProcessingEngine):
+    def __init__(self):
+        self.calls: List[List[Any]] = []  # type: ignore
+        self.values: Dict[str, List[Any]] = {}
```

Comment:
use a defaultdict(list) here so you don't have to worry about initializing all of them (unless you require a keyerror on unkown keys but it does not look like it)